backend/src/main/java/gov/cdc/usds/simplereport/db/model/auxiliary/SupportedDiseaseTestResult.java

```diff
@@ -0,0 +1,12 @@
+package gov.cdc.usds.simplereport.db.model.auxiliary;
+
+import gov.cdc.usds.simplereport.db.model.SupportedDisease;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class SupportedDiseaseTestResult {
```

Comment:
General question. Why is this class to represent a disease result introduced in this PR?. We are already returning disease results for other endpoints so why is this new one introduced?